Jenkinsfile

```diff
@@ -48,6 +48,10 @@ install_tabular_all = """
     python3 -m pip install --upgrade -e tabular/[all,tests]
 """
 
+install_multimodal = """
+    python3 -m pip install --upgrade -e multimodal/[tests]
+"""
+
```

Comment:
You have to resubmit this PR under an awslabs branch or else Jenkinsfile updates won't take effect on the PR to my understanding.